tests/alembic_migrations/versions/v_7b139906ac46_add_not_null_constraint_on_meta_vars/test_smoke.py

```diff
@@ -0,0 +1,84 @@
+"""Smoke tests:
+- Upgrade adds not null constraint
+- Downgrade drops not null constraint
+"""
+
+# -*- coding: utf-8 -*-
+import logging
+import pytest
+from alembic import command
+from sqlalchemy import inspect, text
+
+logger = logging.getLogger("tests")
+
+@pytest.mark.parametrize(
+        "prepared_schema_from_migrations_left", ("3d50ec832e47",),indirect=True
+)
+@pytest.mark.usefixtures("new_db_left")
+def test_upgrade(
+    prepared_schema_from_migrations_left, alembic_config_left, schema_name
+):
+    """test migration from 3d50ec832e47 to 7b139906ac46. """
+    
+    # Set up database to version 7b139906ac46
+    engine, script = prepared_schema_from_migrations_left
+
+    engine.execute(
+        f"INSERT INTO {schema_name}.meta_vars(vars_id)"
+        f"VALUES (1)"
+    )
+
+    command.upgrade(alembic_config_left, "+1")
+
+    # Check that data has been modified to remove nulls
+    result = engine.execute(
+        f"SELECT * FROM {schema_name}.meta_vars"
+    )
+    for row in result:
```

Comment:
If the database starts out empty, you can assume that only the one row you inserted comes back. Try

```
row = next(result)
```

instead of the for loop.

And then you can check `assert row == (....)` where the parens are a tuple.